spec/index.bs

```diff
@@ -2455,13 +2464,31 @@ dictionary GPUTextureCopyView {
 };
 </script>
 
-A {{GPUTextureCopyView}} is a view of a sub-region of a [=texture=] [=subresource=] with the initial offset {{GPUOrigin3D}} in
-texels, used when copying data from or to a {{GPUTexture}}.
-
-  - For {{GPUTextureDimension/2d}} textures, data is copied from or to one [=mipmap level=] and one [=array layer=] of the [=texture=].
+A {{GPUTextureCopyView}} is a view of a sub-region of one or multiple contiguous [=texture=] [=subresources=] with the initial
```

Comment:
we may need to put an Issue here to clarify what "contiguous" means.
I.e. if I select mipmaps 1-2 and array layers 1-2, is that sub-region contiguous?